tex/artikel/Gleichstellungsbuero_zentral.tex

```diff
@@ -6,11 +6,17 @@ \subsection{Wer hilft bei Fällen von Sexismus?}
+Vielleicht ein kurzer Satz zu mir: Ich bin Franzi, 23 Jahre alt und studiere schon etwas länger an der Uni Münster; meine Fächer sind Latein und Judaistik. Seit einem Jahr bin ich außerdem Gleichstellungsbeauftragte für die Statusgruppe der Studierenden, also euch!
 
-Solltet ihr also ein Anliegen haben, bei dem ihr Unterstützung oder ein offenes Ohr braucht, schreibt mir gerne eine E-Mail an \email{franziskajuergens@uni-muenster.de}! 
+Das Gleichstellungsbüro kümmert sich aber nicht nur um die Fälle, die oben geschildert werden – zum Aufgabenbereich gehört unter anderem auch die Beratung bei Einstellungsverfahren in der Lehre oder die Förderung von jungen Frauen in den Wissenschaften – zum Beispiel als Kontrollinstanz bei der Vergabe von Stipendien. 
+Zwar ist in der gesetzlichen Grundlage für die Aufgaben des Gleichstellungsbüros nur von Frauen die Rede, mir ist es aber ein großes Anliegen, Wege zu finden, wie auch nicht binären, trans, inter und agender Personen im Alltag geholfen werden kann. 
+
+Diese Unterstützung kann verschiedene Formen annehmen. Mein momentanes Hauptanliegen ist das konsequente Vorgehen gegen sexualisierte Gewalt. Dazu gehört vor Allem, die verwirrenden Strukturen, die in solchen Fällen tätig werden (können), transparent aufzuschlüsseln und generell mehr Aufmerksamkeit für dieses oft totgeschwiegene Thema zu erzeugen. 
+Aber auch das Studieren mit Kind birgt Herausforderungen, bei denen ihr Unterstützung bekommen könnt: Ich helfe euch gerne dabei, zum Beispiel einen KiTa-Platz oder eine Möglichkeit zur Betreuung in den „Randzeiten“ (also früh morgens oder spät abends) zu finden. Im Studi-Kidz-Café habt ihr außerdem die Möglichkeit, euch mit anderen Eltern, die gerade studieren, auszutauschen. 
+%Während des Studienalltags kommt es häufig zu Situationen, an denen man einfach nicht weiterkommt: seien es Formulare, die Prüfungsvorbereitung oder der ganz normale Uni-Stress. Um diese Momente etwas einfacher zu machen, möchte ich in meiner Amtszeit als stellvertretende zentrale Gleichstellungsbeauftragte aus dem Bereich der Studierenden gerne ein Projekt anstoßen, bei dem Studis aus höheren Semestern den "Neuen" als Buddies, also ganz auf Augenhöhe und informell, zur Seite stehen. 
+%Außerdem ist es eine Herzensangelegenheit von mir, alle Menschen, die sich als Frauen identifizieren, in meiner Arbeit mit einzubeziehen. 
+%Diese Unterstützung kann ganz verschiedene Formen annehmen: Das Studieren mit Kind zum Beispiel birgt Herausforderungen, bei denen euch geholfen werden kann: Ich helfe euch gerne dabei, zum Beispiel einen KiTa-Platz oder eine Möglichkeit zur Nachmittagsbetreuung zu finden. Im Studi-Kidz-Café habt ihr außerdem die Möglichkeit, euch mit anderen Eltern, die gerade studieren, auszutauschen. 
```

Comment:
Was hat es mit den auskommentierten Feldern hier auf sich? Haben die platztechnisch nicht mehr gepasst (dann würde ich den ganzen Text lieber eine Schriftgröße kleiner nehmen), oder sind die veraltet (dann sollten die hier weg)?

Reply:
Das ist veraltet(^^') 
Habe die einfach so drin gelassen (ist bei manchen Artikeln auch so)

Reply:
Würde das dann löschen, sonst hat man in ein paar Jahren viel alten Kram (genau um das trotzdem archiviert zu haben, nutzt die Fibel ja Git)

Reply:
Verstehe, mache ich dann